benchmarks/rust.suite

```diff
@@ -0,0 +1,16 @@
+# Rust benchmarks - all benchmarks written in Rust.
+#
+# This suite includes both existing and newly added Rust benchmarks,
+# covering various workloads from cryptography to serialization to HTML processing.
+
+blake3-scalar/benchmark.wasm
+blind-sig/benchmark.wasm
+hex-simd/benchmark.wasm
+# image-classification/image-classification-benchmark.wasm
+pulldown-cmark/benchmark.wasm
+regex/benchmark.wasm
+rust-compression/benchmark.wasm
+rust-html-rewriter/benchmark.wasm
+rust-json/benchmark.wasm
+rust-protobuf/benchmark.wasm
+# tract-onnx-image-classification/benchmark.wasm
```

Comment:
Why are there these two commented out entries?

Reply:
Looks like `image-classification` requires OpenVINO and doesn't have a .wasm checked into tree.  The onnx test is pure guest execution using tract and is included.  I removed the image-classification bench comment and uncommented tract-onnx here.

Reply:
> Looks like `image-classification` requires OpenVINO and doesn't have a .wasm checked into tree.

Ah okay, I didn't realize that. We should either remove it from the suite or fix the build as necessary check in the wasm binary.